keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepositoryTests.kt

```diff
@@ -62,7 +62,7 @@ internal class SqlVerificationRepositoryTests :
     )
 
   override fun VerificationContext.setup() {
-    artifactRepository.register(artifact)
+    deliveryConfig.artifacts.forEach(artifactRepository::register)
```

Comment:
This change makes it persist all artifacts in the delivery config rather than just the one we know we're interested in